checks/evaluation/branch_protection_test.go

```diff
@@ -49,12 +50,13 @@ func TestIsBranchProtected(t *testing.T) {
 		expected        scut.TestReturn
 	}{
 		{
+			// TODO: Naming is inconsistent. We should change this to make it score 0.
 			name: "Nothing is enabled",
```

Comment:
Hmm makes more sense now. But I still think it should be reviewed, because:
1. The test right below this one is named `Nothing is enabled and values are nil` and in this test I think "nothing is enabled" actually means "they have no branch protection and are scoring 0", so it's not really coherent.
2. The test is assuming the branch requires PRs, but the default doesn't

That said, I'll just remove that comment of mine and change the test name to something clearer like "configs as they are right after creating new Branch Protection setting". Let me know if you don't agree